airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/components.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from dataclasses import InitVar, dataclass
+from typing import Any, Mapping
+
+from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
+from airbyte_cdk.sources.declarative.schema.json_file_schema_loader import JsonFileSchemaLoader, _default_file_path
+
+
+@dataclass
+class BambooHRSchemaLoader(JsonFileSchemaLoader):
```

Comment:
This would benefit from a comment explaining why Bamboo needs a custom schema loader.